src/actions/AuthActions.js

```diff
@@ -578,17 +578,18 @@ export const importAccount = ({ accountName, wif, password }) =>
  */
 export const importSelectedAccounts = (password, accounts) => async (dispatch, getState) => {
 	const wif = getState().form.getIn([FORM_SIGN_IN, 'wif']).value;
-
-	accounts.forEach((account) => {
-		if (account.checked) {
-			dispatch(authUser({
-				accountName: account.name,
+	const accs = accounts.toJS();
+	for (let i = 0; i < accs.length; i += 1) {
```

Comment:
better will be use array function like
accs.filter().map() and then forward to Promise.all

Reply:
Here must be await in this loop, because we need to wait previous async operation with storage before starting next